### ISSUE TITLE
Compatible with Sharship

### ISSUE THREAD

**Issue by GreyElaina** [microsoft/inshellisense#67]
**Is your feature request related to a problem? Please describe.**
when inshellisense enabled, starship's prompt disappeared.

**Describe the solution you'd like**
keep both inshellisense's great completion support and beautiful starship's useful prompt.

**Describe alternatives you've considered**
i donot have any alternative.

**Additional context**
mainly in MacOS, but it could be reproduct on my linux terminal and pd windows vm

**Comment by GreyElaina** [comment]
hmmm...arrow keys not work btw.

**Comment by Souliboi** [comment]
Same as https://github.com/microsoft/inshellisense/issues/10

**Comment by JLHasson** [comment]
yeah, unfortunately cannot start using Inshellisense until Starship is supported. quick view of how the terminal changes from my perspective w/ & w/o inshellisense running:
```bash
apps  🌱 main via  v20.2.0
❯
(base)
apps  🌱 main via  v20.2.0
❯ is
> git co
git: 'co' is not a git command. See 'git --help'.

The most similar commands are
	commit
	clone
	log
> exit
(base)
apps  🌱 main via  v20.2.0
❯
```

**Comment by cpendery** [comment]
Closing as this was fixed in [0.0.1-rc.5](https://github.com/microsoft/inshellisense/releases/tag/0.0.1-rc.5)
